Solutions/Endjin.SemVer.DotNetApi.Specs/Endjin/SemVer/DotNetApi/Specs/LibraryChangeTypeDetection.feature.cs

```diff
@@ -1,8 +1,8 @@
 ﻿// ------------------------------------------------------------------------------
```

Comment:
We should no longer be checking `.feature.cs` files into source control, because since v3, SpecFlow has generated these dynamically as part of the build.

It used to be necessary to have them in here because they were generated by the VS extension, and CI builds wouldn't have access to that. But SpecFlow fixed that some time ago. They're now generated by a custom target added by one of the SpecFlow NuGet packages, so these files can be generated on the build agents.

Leaving these in the repo creates unnecessary noise in every commit that changes anything at all about a test. What you need to do is this:

1. if the `.gitignore` doesn't already contain an entry for `*.feature.cs`, add one
1. delete all the `.feature.cs` files in Solution Explorer. **DO NOT BUILD AT THIS POINT**
1. go into the Git Changes window and commit the deletions

You can now build again and all the `.feature.cs` files will reappear, but they will no longer be in the repo.